scripts/unity/header_prepare.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python
```

Comment:
Zephyr uses python3, so I guess this is safer:
```suggestion
#!/usr/bin/env python3
```
as some systems may have python pointing to `python 2.x` although I do know when called through CMake the specific python interpreter is used.